builder.md

```diff
@@ -1,58 +1,118 @@
 # Builder Guidelines
 **Version 0.10.1**
 
-A base16 builder is an application that can build syntax highlighting definition files for text editors by using base16 scheme files which contain a collection of colours and base16 template files which contain syntax highlighting rules.
+*The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD",
+"SHOULD NOT", "RECOMMENDED",  "MAY", and "OPTIONAL" in this document are to be
+interpreted as described in [RFC 2119](https://datatracker.ietf.org/doc/html/rfc2119).*
 
-Builders are designed for theme maintainers' ease of use. Theme maintainers should provide built versions of their theme so the end user doesn't need to be aware of the builder.
+A base16 builder is an application that can build syntax highlighting definition files for text editors by using base16 scheme files (which contain a collection of colors) and base16 template files (which contain syntax highlighting rules).
```

Comment:
Isn't this a bit narrow?  We support Terminal and other non-editors, do we not?

Reply:
Yes, good point - do we have a term for this yet? "application settings (and other support files)" covers quite a bit more, but is also a bit vaguer than I'd like.

Reply:
I kind of liked "application theming" - the key word being theming, not themes -  but not if that just leads back to using "theme" everywhere...

IE:

>A base16 builder is an application that builds application specific themeing configurations.  It does this by using base16 scheme files (containing a collection of colors) and base16 template files (instructions concerning how to build the application specific files).

Reply:
That sounds good to me!